VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -5377,20 +5378,16 @@ public string DeathLocationTypeHelper
                     Dictionary<string, string> age = new Dictionary<string, string>();
                     Quantity quantity = (Quantity)AgeAtDeathObs.Value;
                     age.Add("value", quantity.Value == null ? "" : Convert.ToString(quantity.Value));
-                    age.Add("unit", quantity.Unit == null ? "" : quantity.Unit);
                     age.Add("code", quantity.Code == null ? "" : quantity.Code);
-                    age.Add("system", quantity.System == null ? "" : quantity.System);
```

Comment:
It may be helpful to continue allowing unit and system to be specified; while not required, I think they are allowed: http://hl7.org/fhir/R4/datatypes.html#Quantity